### PR TITLE
feat: configurable labels

### PR DESCRIPTION
**Pull request by superdyzio**
implements #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,18 @@ Chosen value will be emitted by `customOutput`.
 #### showLetters
 Boolean, default `true`. Shows the letters on top (Y, M, W, D, H, M, S)
 
+#### labels
+Object, accepting labels that should be shown above the fields instead of default letters (`showLetters` have to be set to true).
```
Comment (FrancescoBorzi) [inline README.md]: has* to be set to true

File: README.md [hunk 2]
```diff
@@ -102,6 +102,18 @@ Chosen value will be emitted by `customOutput`.
...
+
+Available fields:
+- years
+- months
+- weeks
+- days
+- hours
+- minutes
+- seconds
```
Comment (FrancescoBorzi) [inline README.md]: maybe add an example?

File: projects/ngx-duration-picker/src/lib/duration-picker.component.html [hunk 3]
```diff
@@ -3,13 +3,13 @@
...
-    <td *ngIf="config.showHours">H</td>
-    <td *ngIf="config.showMinutes">M</td>
-    <td *ngIf="config.showSeconds">S</td>
+    <td id="dp-years-label" *ngIf="config.showYears">{{config.labels.years}}</td>
+    <td id="dp-months-label" *ngIf="config.showMonths">{{config.labels.months}}</td>
+    <td id="dp-weeks-label" *ngIf="config.showWeeks">{{config.labels.weeks}}</td>
+    <td id="dp-days-label" *ngIf="config.showDays">{{config.labels.days}}</td>
+    <td id="dp-hours-label" *ngIf="config.showHours">{{config.labels.hours}}</td>
+    <td id="dp-minutes-label" *ngIf="config.showMinutes">{{config.labels.minutes}}</td>
+    <td id="dp-seconds-label" *ngIf="config.showSeconds">{{config.labels.seconds}}</td>
```
Comment (FrancescoBorzi) [inline projects/ngx-duration-picker/src/lib/duration-picker.component.html]: codestyle: spaces around {{ and }}

File: projects/ngx-duration-picker/src/lib/duration-picker.component.spec.ts [hunk 4]
```diff
@@ -295,6 +295,19 @@ describe('DurationPickerComponent', () => {
     set(component, 1, 0, 0, 0, 0, 0, 0, false, '', 'seconds');
     expect(component.valueInMilliseconds).toEqual(yearInMilliseconds);
   });
+
+  it('specific labels should be overwritten', () => {
+    set(component, 0, 0, 0, 0, 0, 0, 1, false, '', '', {years: 'years', weeks: 'weeks', hours: 'hours'});
```
Comment (FrancescoBorzi) [inline projects/ngx-duration-picker/src/lib/duration-picker.component.spec.ts]: also here

File: README.md (hunk 2 above)
Comment (superdyzio) [reply README.md]: example with polish abbrevietions for weeks and hours added 🎉

**Comment by superdyzio** [comment]
@FrancescoBorzi all comments resolved

I've also had some issues with running e2e locally - some problems with creating the session, because of wrong Chrome version - have you ever had that?

**Comment by FrancescoBorzi** [comment]
@superdyzio not really, have you tried to update your Chrome to the latest version?

**Comment by superdyzio** [comment]
@FrancescoBorzi trying right now
before I needed to add some `postinstall` script which was updating `webdriver-manager` and then it worked

**Comment by FrancescoBorzi** [comment]
something wrong with the travis build

**Comment by superdyzio** [comment]
yeah, I was expecting some changes in `package-lock`, but there is none, so probably not mine fault (all e2e passed locally)

**Comment by superdyzio** [comment]
travis failed again, it requires chrome version 78 😱 
I have ` 78.0.3904.70` locally and it works

**Comment by FrancescoBorzi** [comment]
@superdyzio mmmh, try to revert the changes to `package-lock.json`

We don't need those changes, right?

**Comment by superdyzio** [comment]
`package-lock.json` changes reverted, I guess it is not crucial 😄

**Pull request merged**